src/config/config.ts

```diff
@@ -97,11 +97,13 @@ export class Config implements IConfig {
   public shell!: string
   public theme?: Theme
   public topicSeparator: ' ' | ':' = ':'
+  public updateConfig!: NonNullable<PJSON.CLI['oclif']['update']>
```

Comment:
if we're doing breaking changes, maybe we could make the types honest?  Like, these `!` are kinda true as long as you've called the static `load` method?  

And TS can't help developers avoid runtime undefineds if we do stuff like that.

noodle on it